src/app/containers/MediaPlayer/index.test.jsx

```diff
@@ -1,106 +1,32 @@
-import React from 'react';
-import { string, shape, node } from 'prop-types';
 import { shouldMatchSnapshot, isNull } from '@bbc/psammead-test-helpers';
-import MediaPlayerContainer from '.';
-import { RequestContextProvider } from '#contexts/RequestContext';
-import { ServiceContextProvider } from '#contexts/ServiceContext';
-import { ToggleContext } from '#contexts/ToggleContext';
-import { validVideoFixture, missingVpidFixture } from './helpers/fixtures';
-
-const defaultToggles = {
-  local: {
-    mediaPlayer: {
-      enabled: true,
-    },
-  },
-  test: {
-    mediaPlayer: {
-      enabled: true,
-    },
-  },
-  live: {
-    mediaPlayer: {
-      enabled: false,
-    },
-  },
-};
-
-const mockToggleDispatch = jest.fn();
-const ContextWrapper = ({ platform, children, toggleState }) => (
-  <RequestContextProvider
-    isAmp={platform === 'amp'}
-    service="news"
-    statusCode={200}
-    platform={platform}
-    id="foo"
-    pageType="article"
-    pathname="/pathname"
-  >
-    <ServiceContextProvider service="news">
-      <ToggleContext.Provider
-        value={{ toggleState, toggleDispatch: mockToggleDispatch }}
-      >
-        {children}
-      </ToggleContext.Provider>
-    </ServiceContextProvider>
-  </RequestContextProvider>
-);
-
-ContextWrapper.propTypes = {
-  children: node.isRequired,
-  platform: string.isRequired,
-  toggleState: shape({}),
-};
-
-ContextWrapper.defaultProps = {
-  toggleState: defaultToggles,
-};
+import {
+  VideoFixture,
+  VideoAmpFixture,
+  VideoFixtureNoPlaceHolder,
+  VideoFixtureNoVersionId,
+  VideoFixtureToggledOff,
```

Comment:
'Fixture' seems redundant info. Also it's not just Video is it? And I think we could be explicit about the canonical ones. What about naming like:
```
MediaPlayerCanonical
MediaPlayerCanonicalNoPlaceholder
MediaPlayerAmp
```
etc.? Open to other suggestions.

Reply:
Before the refactor there were only test for the video player and not the audio player so I kept it consistent. Was thinking of creating a new issue to add them in.